pommerman/envs/graphics.py

```diff
@@ -0,0 +1,175 @@
+from gym.envs.classic_control import rendering
+from gym.utils import reraise
+import numpy as np
+import os
+
+try:
+    import pyglet
+except ImportError as e:
+    reraise(suffix="HINT: you can install pyglet directly via 'pip install pyglet'. But if you really just want to install all Gym dependencies and not have to think about it, 'pip install -e .[all]' or 'pip install gym[all]' will do it.")
+
+try:
+    from pyglet.gl import *
+except ImportError as e:
+    reraise(prefix="Error occured while running `from pyglet.gl import *`",suffix="HINT: make sure you have OpenGL install. On Ubuntu, you can run 'apt-get install python-opengl'. If you're running on a server, you may need a virtual frame buffer; something like this should work: 'xvfb-run -s \"-screen 0 1400x900x24\" python <your_script.py>'")
+
+# A map between the np value (Item(Enum) class)
+# to the position in the sprite array (ImageGrid).
+#
+# prob a better way to do this...
```

Comment:
like with a dictionary? And if you use the symbols for keys, it also becomes obvious what is being mapped.